cvpy/annotation/cvat/CVATProject.py

```diff
@@ -149,20 +150,163 @@ def post_images(self, image_table: ImageTable) -> None:
         # Images successfully posted for this task, add it to the projects task list.
         self.add_task(task)
 
-    def get_annotations(self, annotated_table: ImageTable, image_table: ImageTable) -> None:
+    def get_annotations(self, image_table: ImageTable, annotated_table: CASTable) -> None:
         """
         Fetch annotations from CVAT corresponding to the images in a CAS table.
 
         Parameters
         ----------
-        annotated_table:
-            Specifies the output table where the images and the corresponding annotations will be stored.
         image_table:
-            Specifies the input table containing encoded images that was used in a call to post_images()
-            on this CVATProject object.
-        
+            Specifies the input ImageTable that was used in a call to post_images() on this CVATProject object.
+        annotated_table:
+            Specifies the output CAS table where the images and the corresponding annotations will be stored.
+
         """
-        pass
+
+        # Determine which unique CVAT task is associated with the input image table.
+        tasks = self.get_tasks()
+        for task in tasks:
+            if task.image_table == image_table:
+                main_task = task
+
+        # Get the tasks from CVAT.
+        task_response = requests.get(f'{self.url}/api/tasks', headers=self.credentials.get_auth_header())
```

Comment:
This API call seems to be fetching ALL tasks in CVAT server since there is no task_id or task_name parameter being passed. Am I missing something? 

Also, see my next comment. This call may not be required.

Reply:
Good catch. I modified this get() to actually use the main_task that was identified. I think it is required to actually get that task information from the image_table that was passed